### PR TITLE
French translation issue #288 - Root folder translations & lesson 2.1

### PR DESCRIPTION
**Pull request by IterateMe**

### CODE REVIEW

**Comment by Charma95** [comment]
Hi Vianney,

Here are the changes that I would make in IoT-For-Beginners/translations/README.fr.md

- Line 16, I would change néophytes for débutant since néophytes is more appropriate in a religious context. https://www.larousse.fr/dictionnaires/francais/n%C3%A9ophyte/54217
- Line 19, I would change ventillé for divisée and I would also change 'l'équipe d'Azure infonuagique' by partisans since advocate and team have a different semantics as per se.
- Line 28, there is a missing traduction in ' Click the image for a larger version.'
- Line 62, I would change 'd'infonuagique'  and there is a small mistake in 'télémètrie' replace it for 'télémétrie'
- The local github links are not working it is probably only a ../ missing at the beginning.

Here are the changes that I would make in IoT-For-Beginners/translations/SECURITY.fr.md : 

- Line 38, 'Policy' hasn't been translated.

Here are the changes that I would make in IoT-For-Beginners/translations/SUPPORT.fr.md : 
- None

Here are the changes that I would make in IoT-For-Beginners/translations/TRANSLATIONS.fr.md:
-None

Here are the changes that I would make in IoT-For-Beginners/2-farm/translations/README.fr.md: 
- The local links are not working line 16 to 21 ../ before should do the trick.

Here are the changes that I would do in IoT-For-Beginners/2-farm/lessons/1-predict-plant-growth/translations/README.fr.md:
- Line 78, the titlte hasn't been translated
- Line 94 GDD refers to the english words, it should be DJC
- Line 105, 'Example' should be written 'Exemple' in french
- Line 130 same as in Line 94 
- Line 136, GDD should be DJC
- Line 205, the python code is not formatted correctly, you can see it in preview mode. 
- Line 231, same as in line 94

Here are the changes that I would make in IoT-For-Beginners/2-farm/lessons/1-predict-plant-growth/translations/assignment.fr.md:
- Line 12, GDD should be DJC 
- Line 37, the image is not showing the link is missing ../ at the beginning.

Here are the changes that I would make in IoT-For-Beginners/2-farm/lessons/1-predict-plant-growth/translations/pi-temp.fr.md:
- None

Here are the changes that I would make in IoT-For-Beginners/2-farm/lessons/1-predict-plant-growth/translations/single-board-computer-temp-publish.fr.md:
-Line 56, there is English remaining at the end of the line.

Here are the changes that I would make in IoT-For-Beginners/2-farm/lessons/1-predict-plant-growth/translations/virtual-device-temp.fr.md:
-Line 2, there is an extra space in the heading markdown

Here are the changes that I would make in IoT-For-Beginners/2-farm/lessons/1-predict-plant-growth/translations/wio-terminal-temp-publish.fr.md:
- None

Here are the changes that I would make in IoT-For-Beginners/2-farm/lessons/1-predict-plant-growth/translations/wio-terminal-temp.fr.md:
-Line 103, 0ème sounds really not familiar i would change it for 'élément 0'

**Comment by IterateMe** [comment]
I implemented the suggested changes, tell me if you see other mistakes, I completed the missing translations from the root folder. Please review the as well!

**Comment by jimbobbennett** [comment]
Thanks @IterateMe!

@Charma95 - let me know how this looks now and if it's ready to merge.

**Comment by Charma95** [comment]
@jimbobbennett  Yes, the translation has been reviewed and corrections have been made according to suggestions and it's now ready to merge.

File: 2-farm/lessons/1-predict-plant-growth/translations/README.fr.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
+[![French](https://img.shields.io/badge/-French-purple)](translations/README.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/lessons/1-predict-plant-growth/translations/README.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: 2-farm/lessons/1-predict-plant-growth/translations/assignment.fr.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+[![French](https://img.shields.io/badge/-French-purple)](assignment.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/lessons/1-predict-plant-growth/translations/assignment.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: 2-farm/lessons/1-predict-plant-growth/translations/pi-temp.fr.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
+[![French](https://img.shields.io/badge/-French-purple)](pi-temp.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/lessons/1-predict-plant-growth/translations/pi-temp.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: 2-farm/lessons/1-predict-plant-growth/translations/single-board-computer-temp-publish.fr.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+[![French](https://img.shields.io/badge/-French-purple)](single-board-computer-temp-publish.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/lessons/1-predict-plant-growth/translations/single-board-computer-temp-publish.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: 2-farm/lessons/1-predict-plant-growth/translations/virtual-device-temp.fr.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
+[![French](https://img.shields.io/badge/-French-purple)](virtual-device-temp.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/lessons/1-predict-plant-growth/translations/virtual-device-temp.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: 2-farm/lessons/1-predict-plant-growth/translations/wio-terminal-temp-publish.fr.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+[![French](https://img.shields.io/badge/-French-purple)](wio-terminal-temp-publish.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/lessons/1-predict-plant-growth/translations/wio-terminal-temp-publish.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: 2-farm/lessons/1-predict-plant-growth/translations/wio-terminal-temp.fr.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
+[![French](https://img.shields.io/badge/-French-purple)](wio-terminal-temp.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/lessons/1-predict-plant-growth/translations/wio-terminal-temp.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: 2-farm/translations/README.fr.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+[![French](https://img.shields.io/badge/-French-purple)](translations/README.fr.md)
```
Comment (jimbobbennett) [inline 2-farm/translations/README.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: translations/SECURITY.fr.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
+[![French](https://img.shields.io/badge/-French-purple)](SECURITY.fr.md)
```
Comment (jimbobbennett) [inline translations/SECURITY.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: translations/SUPPORT.fr.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+[![French](https://img.shields.io/badge/-French-purple)](SUPPORT.fr.md)
```
Comment (jimbobbennett) [inline translations/SUPPORT.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

File: translations/TRANSLATIONS.fr.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
+[![French](https://img.shields.io/badge/-French-purple)](TRANSLATIONS.fr.md)
```
Comment (jimbobbennett) [inline translations/TRANSLATIONS.fr.md:1]: Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

**Comment by jimbobbennett** [comment]
Thanks @IterateMe. @Charma95 - are you happy for me to merge this?

**Comment by IterateMe** [comment]
Yes! For me, this is my first contribution to an open source project. Any hint on how to improve the value of my contribution is more than welcome!
Thank you!

**Review by jimbobbennett — APPROVED** [review]

**Comment by jimbobbennett** [comment]
Thanks for your contribution @IterateMe, C'est très bon! This is great work, and is of great value to any French speakers who want to learn IoT. I appreciate how you took on board the feedback and were happy to adhere to our format and guidelines.

Nothing extra I can thin for you to do to improve the value except keep up the great work!

Merci beaucoup!

**Pull request merged**